nutshell.js

```diff
@@ -434,6 +434,33 @@ Bubble: the box that expands below an expandable, containing a Nutshell Section
             // What punctuation (in this language) signifies the END of a sentence? Note, this is a regex.
             endPunctuation: /[。.?!]\s/g // added chinese period
 
+        },
+        he: {
+            // Button text
+            closeAllNutshells: `סגור את כל האגוזים`,
+            learnMore: `עוד אודות קליפת האגוז`,
```

Comment:
Might be better to keep the original English name (nutshell), rather than translate it (קליפת האגוז).